unit/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -119,6 +119,18 @@ TEST_CASE("\"symbol_exprt\" to smt term conversion", "[core][smt2_incremental]")
     smt_identifier_termt("foo", smt_bool_sortt{}));
 }
 
+TEST_CASE("\"nondet_symbol_exprt\" to smt term conversion", "[core][smt2_incremental]")
+{
+    auto test = expr_to_smt_conversion_test_environmentt::make(test_archt::i386);
+    // Checking that the nondet_symbol is translated correctly to a term lookup
+    CHECK(
+            test.convert(nondet_symbol_exprt{"nondet_symbol1", bool_typet{}}) ==
+            smt_identifier_termt("nondet_symbol1", smt_bool_sortt{}));
+    CHECK(
+            test.convert(nondet_symbol_exprt{"nondet_symbol2", bitvector_typet{ID_bv, 42}}) ==
```

Comment:
:no_entry_sign: See my other comment on the use of `ID_bv`.

Reply:
Changed